data_model/src/events/data/events.rs

```diff
@@ -417,7 +440,7 @@ impl Event {
             | Self::Configuration(_)
             | Self::Role(_)
             | Self::PermissionToken(_)
-            | Self::PermissionValidator(_) => None,
+            | Self::Validator(_) => None,
```

Comment:
what is the reason for rename?

Reply:
Oh, actually no. I just removed it at first, and bring back later. Will rename back to `PermissionValidator`